linux/helper_centos.sh

```diff
@@ -6,7 +6,7 @@ add_repo() {
     if [ "$STAGING_REPO" = "true" ]; then
         repo="http://nr-downloads-ohai-staging.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/yum/el/8/x86_64/newrelic-infra.repo"
     else
-        repo="https://download.newrelic.com/infrastructure_agent/linux/yum/el/8/x86_64/newrelic-infra.repo"
+        repo="http://nr-downloads-main.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/yum/el/8/x86_64/newrelic-infra.repo"
```

Comment:
I am not sure if this will work as we expect it to. If you curl the file, you can see it has hardcoded the URL that points to fastly:
```
curl http://nr-downloads-main.s3-website-us-east-1.amazonaws.com/infrastructure_agent/linux/zypp/sles/12.4/x86_64/newrelic-infra.repo
[newrelic-infra]
name=New Relic Infrastructure
baseurl=https://download.newrelic.com/infrastructure_agent/linux/zypp/sles/12.4/x86_64
gpgkey=https://download.newrelic.com/infrastructure_agent/gpg/newrelic-infra.gpg
gpgcheck=1
repo_gpgcheck=1
```
Same goes for suse. Ubuntu should work.

To hack these two we'd likely need to `sed` the old url with the new one.

Reply:
Locally it worked, I guess that the cache was updated by chance then